pyproject.toml

```diff
@@ -47,6 +47,10 @@ sarimax = [
     "statsmodels>=0.12, <0.15",
 ]
 
+rnn = [
+    "tensorflow>=2.13.0"
```

Comment:
Should we name this dependence deep_leerning in case we extend it to more models in the future?
We also need to fix the upper version.

Reply:
I initially thought that RNN was generic enough, but there are indeed other deep learning models to do forecasting that may come in the future ;)

Reply:
Done in commit [requirements deep_learning](https://github.com/JoaquinAmatRodrigo/skforecast/pull/581/commits/aef17c3345a84b2f60c8f7f9c903ba32e70f8cb2)